openverse_catalog/dags/providers/provider_api_scripts/provider_data_ingester.py

```diff
@@ -163,7 +166,13 @@ def get_batch(self, query_params: Dict) -> Tuple[Optional[List], bool]:
             # this will return True and ingestion continues.
             should_continue = self.get_should_continue(response_json)
 
-        except Exception as e:
+        except (
+            RequestException,
+            RetriesExceeded,
+            JSONDecodeError,
+            ValueError,
+            TypeError,
+        ) as e:
             logger.error(f"Error due to {e}")
```

Comment:
Would something like "Error getting next query parameters due to {e}" be clearer?

Reply:
Great idea!